new_platforms/tests/oetests_host/OEHostTest.cpp

```diff
@@ -122,23 +122,24 @@ class OEHostTest : public TrustedAppTest {
 
 TEST_F(OEHostTest, get_report_v1_Success)
 {
-    uint8_t report_buffer[1024];
+    uint8_t report_buffer[4096];
     size_t report_buffer_size = sizeof(report_buffer);
 
     oe_result_t oeResult = oe_get_report_v1(GetOEEnclave(),
                                             0,
                                             NULL, // opt_params,
                                             0,    // opt_params_size,
                                             report_buffer,
-                                            &report_buffer_size);
+                                            &report_buffer_size); // THIS ONE WILL HAPPILY OVERFLOW
```

Comment:
No longer the case after Dave's oeedger8r fix.

Reply:
Right, remove comment

Reply:
Removed in the latest version.